Philips.CodeAnalysis.Test/Maintainability/Documentation/XmlDocumentationShouldAddValueAnalyzerTest.cs

```diff
@@ -0,0 +1,283 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Philips.CodeAnalysis.Common;
+using Philips.CodeAnalysis.MaintainabilityAnalyzers.Documentation;
+
+namespace Philips.CodeAnalysis.Test.Maintainability.Documentation
+{
+	[TestClass]
+	public class XmlDocumentationShouldAddValueAnalyzerTest : CodeFixVerifier
+	{
+		#region Non-Public Data Members
+
+		private const string configuredAdditionalUselessWords = "dummy,tummy";
+
+		#endregion
+
+		#region Non-Public Properties/Methods
+
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => new XmlDocumentationShouldAddValueAnalyzer();
+
+		protected override CodeFixProvider GetCSharpCodeFixProvider()
+		{
+			return new XmlDocumentationCodeFixProvider();
+		}
+
+		protected override Dictionary<string, string> GetAdditionalAnalyzerConfigOptions()
+		{
+			Dictionary<string, string> options = new Dictionary<string, string>
+			{
+				{ $@"dotnet_code_quality.{ Helper.ToDiagnosticId(DiagnosticIds.XmlDocumentationShouldAddValue) }.additional_useless_words", configuredAdditionalUselessWords  }
+			};
+			return options;
+		}
+
+		#endregion
+
+		#region Public Interface
+		[TestMethod]
+		public void EmptyClassTest()
+		{
+			string content = $@"
+/// <summary></summary
+public class Foo
+{{
+}}
+";
+
+			VerifyCSharpDiagnostic(content, DiagnosticResultHelper.CreateArray(DiagnosticIds.EmptyXmlComments));
+		}
+
+		[TestMethod]
+		public void EmptyMethodTest()
+		{
+			string content = $@"
+public class TestClass
+{{
+	/// <summary></summary
```

Comment:
I don't think it catches this:

```
/// <summary>
/// 
/// </summary>
```

That's the default I believe.

Always maybe confirm this is caught:
```
/// <summary>
/// 
/// </summary>
/// <returns></returns>
```